catalog/app/containers/Bucket/PackageTree.js

```diff
@@ -300,7 +301,17 @@ function DirDisplay({ bucket, name, revision, path, crumbs }) {
       }))
       return (
         <>
-          <TopBar crumbs={crumbs} />
+          <TopBar crumbs={crumbs}>
+            {hashData.case({
+              Ok: ({ hash }) => (
+                <FileView.ZipDownloadForm
+                  label="Download package as .zip"
```

Comment:
Same as above, use .zip/archive icon
```suggestion
                  label="Download package"
```